### ISSUE TITLE
Installation Error pygraphviz 1.3

### ISSUE THREAD

**Issue by papachoco** [pygraphviz/pygraphviz#69]
Hi... I am getting the error below while installing pygraphviz 1.3 using easy-install / buildout

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 2245, in main
    
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 380, in run
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 610, in easy_install
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 659, in install_item
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 842, in install_eggs
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 1070, in build_and_install
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/command/easy_install.py", line 1056, in run_setup
    
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 240, in run_setup
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 193, in setup_context
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 164, in save_modules
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 139, in resume
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 152, in save_modules
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 193, in setup_context
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 237, in run_setup
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 267, in run
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 236, in runner
  File "build/bdist.macosx-10.10-x86_64/egg/setuptools/sandbox.py", line 46, in _execfile
  File "/var/folders/jd/g7fjfbn57695542zy0j_84b40000gn/T/easy_install-sXVhNC/pygraphviz-1.3/setup.py", line 16, in <module>
ImportError: No module named setup_commands
An error occurred when trying to install pygraphviz 1.3. Look above this message for any errors that were output by easy_install.
While:
  Installing eggs.
  Getting distribution for 'pygraphviz==1.3'.
Error: Couldn't install: pygraphviz 1.3

**Comment by hagberg** [comment]
Looks like https://github.com/pygraphviz/pygraphviz/blob/master/setup_commands.py
didn't make it into the manifest.  MANIFEST.in needs to be updated and a new release cut.

**Comment by jodal** [comment]
Hoping this can be fixed quickly, as it breaks the docs build for projects using pygraphviz in their Sphinx docs at readthedocs.org.

**Comment by jodal** [comment]
Running `check-manifest` on a fresh Git clone of this repo:

```
lists of files in version control and sdist do not match!
missing from VCS:
  pygraphviz/version.py
missing from sdist:
  LICENSE
  Makefile
  doc/source/static/contents.png
  doc/source/static/navigation.png
  pygraphviz/graphviz.i
  pystrings.swg
  setup_commands.py
  tox.ini
suggested MANIFEST.in rules:
  include *.py
  include *.swg
  include LICENSE
  include Makefile
  include tox.ini
  recursive-include doc *.png
  recursive-include pygraphviz *.i
```

**Comment by hagberg** [comment]
I think I got this fixed in 1.3.1.  Let me know if there are any other issues.

**Comment by erichuanggit** [comment]
Thanks hagberg.
